x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/analytics_logic.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { kea, MakeLogicType } from 'kea';
+import queryString from 'query-string';
+
+import { KibanaLogic } from '../../../shared/kibana';
+import { HttpLogic } from '../../../shared/http';
+import { flashAPIErrors } from '../../../shared/flash_messages';
+import { EngineLogic } from '../engine';
+
+import { AnalyticsData, QueryDetails } from './types';
+
+export interface AnalyticsValues extends AnalyticsData, QueryDetails {
```

Comment:
Do we need to export this, and `AnalyticsActions` below?

Reply:
oh snap, good catch, thanks! I left this in when copying over the original logic file 🤦‍♀️

Reply:
49c6faf